Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -25,4 +25,13 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     const double initialStateOfChargePercent_;
 
     // Add your necessary variables here
+    double voltage_;
+    double current_;
+    double hour_;
+    double ampHour_;
```

Comment:
I would specify that this is the total ampHour to explicitly differentiate it from the initialamphour